c2corg_ui/static/js/editing/imageediting.js

```diff
@@ -51,6 +51,11 @@ app.ImageEditingController = function($scope, $element, $attrs, $http,
    */
   this.exposureError = false;
 
+  /**
+   * @type {?string}
+   * @export
+   */
+  this.initial_image_type = null;
```

Comment:
I wanted to be consistent with the attribute in the model which is ``image_type``.